data/overlayfiles/motd-manager-server-byos/etc/motd

```diff
@@ -0,0 +1,19 @@
+SUSE Manager {SUMA_VERSION} Server
+{OS_PRETTY_NAME} {ARCH} (64-bit)
+
+    ____
+  /@    ~-.
+  \/ __ .- |
+   // // @
+
+Before setting up SUSE Manager, please make sure the system's host name
+is configured to match its DNS name.
+
+To setup SUSE Manager please execute: sudo /sbin/yast2 susemanager_setup
+
+{INCLUDES}
+Documentation: https://www.suse.com/documentation/suse_manager/
+BYOS Guide: https://documentation.suse.com/suma/{SUMA_VERSION}/en/suse-manager/specialized-guides/qs-publiccloud/overview.html
```

Comment:
That URL only seems to work for 4.3 and not for 4.2. https://documentation.suse.com/suma/{SUMA_VERSION}/en/suse-manager/quickstart-public-cloud/qs-publiccloud-overview.html works for both. But I just realized that SUSE Manager 4.2 has  only a couple of weeks of support left so perhaps this is a bit of a moot point anyway. ;)